### PR TITLE
[6572] Dodano sortowanie alfabetyczne dla dropdownu

### PR DESCRIPTION
**Pull request by wiktorcie**
https://bugzilla.stxnext.pl/show_bug.cgi?id=6572

### CODE REVIEW

File: src/presence_analyzer/views.py [hunk 1]
```diff
@@ -38,7 +38,17 @@ def users_view(user_id=None):
     Users listing for dropdown.
     """
     data = get_data()
-    result = assign_ids_to_names_from_xml(data, user_id)
+    assigned_data = assign_ids_to_names_from_xml(data, user_id)
+    result = []
+
+    if len(assigned_data) == 2:
```
Comment (mjelonek92) [inline src/presence_analyzer/views.py]: What is this magic 2? =)

File: src/presence_analyzer/views.py [hunk 2]
```diff
@@ -38,7 +38,17 @@ def users_view(user_id=None):
...
+            return assigned_data
+    else:
+        for k,v in assigned_data.iteritems():
+            v['id'] = k
```
Comment (mjelonek92) [inline src/presence_analyzer/views.py]: I'm confused. What's the purpose of that?

File: src/presence_analyzer/views.py [hunk 3]
```diff
@@ -38,7 +38,17 @@ def users_view(user_id=None):
...
+            result.append(v)
+
+        result = sorted(result, key = lambda k: k['name'])
```
Comment (mjelonek92) [inline src/presence_analyzer/views.py]: No spaces around `=`.

File: src/presence_analyzer/views.py (hunk 2 above)
Comment (wiktorcie) [inline src/presence_analyzer/views.py]: result = sorted(result, key = lambda k: k['name'])

Sorted doesn't accept dicts, so I had to change it into properly formatted list.

File: src/presence_analyzer/views.py (hunk 1 above)
Comment (wiktorcie) [inline src/presence_analyzer/views.py]: Dunno, seems to work though... Just kidding, it means we only have a request for a single user. In this case it's from {"image": "https://intranet.stxnext.pl:443/api/images/users/11", "name": "Maciej D."}

**Pull request merged**